scripts/snyk-containers-scan/run.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# exit when any command fails
+set -e
+
+if [ -z "${SNYKTOKEN}" ]; then
+    echo -e "Could not find snyk token, skipping scan"
+    exit 0
+fi
+
+repo_dir=$GITHUB_WORKSPACE
+file_to_search=Dockerfile
+tmp_dir="${repo_dir}/tmp.${RANDOM}"
+severity_threshold=critical
+docker_workspace=/opt/workspace/
+repo_name="$(basename "$repo_dir")"
+timestamp=$(date +%s)
+stdout_file=${repo_name}.${timestamp}
+pull_number=$(jq --raw-output .pull_request.number "$GITHUB_EVENT_PATH")
+PR_URL="$GITHUB_API_URL/repos/$GITHUB_REPOSITORY/pulls/$pull_number/files"
+
+mkdir -p $tmp_dir
+
+if [ -z "${GITHUB_BASE_REF}" ] ; then
```

Comment:
@davidsalvador-tf is this what you meant ?? if i detect a push, i do not fail, but instead i check if there is at least a dockerfile at the root directory